Cookbook/Technical-Documents/WhatIDontKnow.md

```diff
@@ -0,0 +1,183 @@
+This is our FAQ section. Every question that you may have have a short answer and/or links to additional resources. If during your onboarding you ever find yourself asking a question that is not on this list, create a new item, find an answer with your buddy and update the board.
+
+#### List of events you should have in your calendar
+- Your squad’s daily/retro/ planning
+- Babylon standup
+- Frontend demo
+- Proposal meetings
+- PR party
+- Lunch and Learn
+- iOS Banter
+- 1-on-1 with your manager
+
+#### List of documents you should star on the Google drive
+
+[iOS Team Plan End 2018 - 2019](https://docs.google.com/spreadsheets/d/1kdY3edy_TeqIGH_7VnZzElxgVo_qD2z4EF-arWNShyw/edit#gid=0)
+
+Why you should read it: This document shows Who, What When in our team
+
+[iOS Goals H2 (July - December)](https://docs.google.com/spreadsheets/d/1nI1RAdsP2nJiUzsvCE9LHQH11-OIPNSHZvpMPUQal_o/edit#gid=1533898989)
+
+Why you should read it: Here you can see our goals not related to daily squad work.
+
+### Jargon translator
+In our team’s daily conversations very often you will encounter strangely looking/sounding terms. This section is to help you get familiar with both our technical jargon and cultural references. Only here you can learn what does :renderable: mean on the Slack :)
+
+#### What is NVL?
+
+This is our New Visual Language, more at: [NVL](https://babylonpartners.atlassian.net/wiki/spaces/V4/pages/946077746/NVL+Project+doc)
+
+Why you should read it: You can’t escape from the NVL so it’s better to know what it is about.
+
+#### What is Octopus?
+
+[Octopus 🐙](https://babylonhealth.slack.com/archives/GD6J0QWQ5/p1548151935005200)
+
+Why you should read it: Maybe not aloud, but absolutely everyone asked himself that question.
+
+#### What are WOWs?
+
+What, Owner, When, more at:
+[WOWs](https://sites.google.com/babylonhealth.com/induction/about-us/wows)
+
+Why you should read it: This is something that will show up in your plannings quite often.
+
+#### What is :renderable: emoji?
+
+It escalated from [this](https://babylonhealth.slack.com/archives/G9S9L0TEK/p1557410403027900)
+to [this](https://babylonhealth.slack.com/archives/G22EWKXQT/p1557428026144000)
+
+Why you should read it: Wait till you see it on the slack :D
+
+### Reading list
+This is the list of docs, wikis, confluence pages and articles that we think you should read to get the context of how and what we are doing.
+
+[iOS Cookbook](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/README.md)
+
+Why you should read it: This is our “Bible”, nothing more to add.
+
+[iOS Cookbook - style guide](https://github.com/Babylonpartners/ios-playbook/tree/master/Cookbook/Style-guide)
+
+Why you should read it: Worth to get familiar with before making your first PR.
+
+[iOS Cookbook - technical documents](https://github.com/Babylonpartners/ios-playbook/tree/master/Cookbook/Technical-Documents)
+
+Why you should read it: Those may answer many of your technical questions.
+
```

Comment:
Maybe add the "[New Hires Checklist](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/NewHiresCheckList.md)" as well? It has useful lists for accounts and Slack channels, and the guide to set up dev environment